index.js

```diff
@@ -51,6 +52,7 @@ module.exports = async (input = 'patch', options) => {
 	const pkgManagerName = options.yarn === true ? 'Yarn' : 'npm';
 	const rootDir = pkgDir.sync();
 	const hasLockFile = fs.existsSync(path.resolve(rootDir, 'package-lock.json')) || fs.existsSync(path.resolve(rootDir, 'npm-shrinkwrap.json'));
+	const isOnGitHub = options.repoUrl && options.repoUrl.includes('github');
```

Comment:
`isOnGitHub` would incorrectly be `true` for GitLab repositories with “github” in their name. I’d suggest [hosted-git-info](https://github.com/npm/hosted-git-info) to reliably detect the repository’s host.